packages/inter-protocol/src/auction/scheduleMath.js

```diff
@@ -24,25 +24,48 @@ export const computeRoundTiming = (params, baseTime) => {
   /** @type {NatValue} */
   const lowestRate = params.getLowestRate();
 
+  const noNextAuction = msg => {
```

Comment:
It's surprising to me that this calculator would take responsibility for handling bad params. I think it would be safer to keep the throws and make each caller recover from bad values. I'm willing to accept this as tech debt if it solves the problem